reference/functions/pgroonga-condition.md

```diff
@@ -84,23 +84,228 @@ Please refer to [Calling Functions][sql-syntax-calling-funcs] for information ab
 
 ## Usage
 
-## See also
+### Specify `index_name`
 
-* [postgres_fdw][postgres-fdw]
+シーケンシャルサーチ実行時でも、インデックスに指定したノーマライザーやトークナイザーのオプションを使って検索する方法を紹介します。
```

Comment:
Introducing how to search with normalizer and tokeniser options indicated in the index  while sequential search is executed.